README.md

```diff
@@ -82,8 +84,18 @@ The specs contained in this repository are:
 **Other/related/included:**
 - [PDD](/pdd-protocol-driven-developement) - Protocol Driven Development
 
-## Collaborating
+## Contribute
 
 Suggestions, contributions, criticisms are welcome. Though please make sure to familiarize yourself deeply with IPFS, the models it adopts, and the principles it follows.
 
 Please be aware that specs are really hard to design by committee. Treat this space like you would the workshop of an artist. Please suggest improvements, but please don't be disappointed if we say no to something. What we leave out is often more important than what we add in.
+
+Feel free to join in. All welcome. Open an [issue](https://github.com/ipfs/specs/issues)!
+
+This repository falls under the IPFS [Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
+
+[![](https://cdn.rawgit.com/jbenet/contribute-ipfs-gif/master/img/contribute.gif)](https://github.com/ipfs/community/blob/master/contributing.md)
+
+## License
+
+[CC-BY-SA](LICENSE)
```

Comment:
Regarding version -- the most current version of CC-BY-SA is 4.0, and it is different than the previous versions. The version should probably be mentioned. Regarding unported-ness, in the 3.0 version, there were "ports" to different countries legal systems (e.g. 

See https://creativecommons.org/faq/#should-i-choose-an-international-license-or-a-ported-license and https://wiki.creativecommons.org/wiki/License_Versions for more detail.